src/src/pages/Metrics/components/SelectForm/SelectForm.tsx

```diff
@@ -147,6 +148,13 @@ function SelectForm({
   if (selectedCount > 0) {
     metricsButtonText += ` (${selectedCount})`;
   }
+
+  let selectedRunHash = useLocation()?.runProps?.hash;
+  let prefilledQuery = '';
+  if (selectedRunHash !== '') {
```

Comment:
maybe `selectedRunHash !== undefined` like this if you go over the metrics tab from the navbar you see `run.hash == "undefined"`